src/gui/transferlistmodel.cpp

```diff
@@ -709,52 +709,46 @@ QColor getDefaultColorByState(const BitTorrent::TorrentState state)
     case BitTorrent::TorrentState::DownloadingMetadata:
     case BitTorrent::TorrentState::ForcedDownloadingMetadata:
         if (!dark)
-            return {34, 139, 34}; // Forest Green
+            return {15, 187, 88}; // Deep Green
         else
-            return {50, 205, 50}; // Lime Green
+            return {108, 194, 74}; // Light Green
     case BitTorrent::TorrentState::StalledDownloading:
     case BitTorrent::TorrentState::StalledUploading:
         if (!dark)
             return {0, 0, 0}; // Black
         else
-            return {204, 204, 204}; // Gray 80
+            return {158, 158, 158}; // Grey
     case BitTorrent::TorrentState::Uploading:
     case BitTorrent::TorrentState::ForcedUploading:
-        if (!dark)
-            return {65, 105, 225}; // Royal Blue
-        else
-            return {99, 184, 255}; // Steel Blue 1
+            return {66, 133, 244}; // Blue
     case BitTorrent::TorrentState::PausedDownloading:
-        return {250, 128, 114}; // Salmon
+        return {242, 80, 34}; // Orange
     case BitTorrent::TorrentState::PausedUploading:
-        if (!dark)
-            return {0, 0, 139}; // Dark Blue
-        else
-            return {79, 148, 205}; // Steel Blue 3
+            return {15, 187, 88}; // Deep Green
```

Comment:
It's used for Downloading in light theme. Pick a color not used somewhere else.

Reply:
I think I did something bad, but check it out. I just couldn't find a yellow color that works on white.

Reply:
I don't see anything bad. I like yellow on dark but... isn't it a flashy color for a state that's supposed to be "this is finished, forget it, let it fade in the background". Maybe i'm nitpicking.

Reply:
>isn't it a flashy color for a state that's supposed to be "this is finished, forget it, let it fade in the background"?

I agree, @now-im please leave colors alone etc. until others provide more feedback & looks like you'll also now have to wait for maintainer approval.

Only thing I can suggest you do is keep re-basing periodically to avoid any conflicts going forward.

Reply:
I will figure something out. After that I am leaving the PR as is.